### ISSUE TITLE
Reduce height of card images/illustrations

### ISSUE THREAD

**Issue by singhinderdeep** [bcgov/digital.gov.bc.ca#544]
#### Problem 
We've had a lot of compliments on the illustrations, however, also some feedback on the ones used on the cards on the homepage: they cause the cards to take up a lot of vertical space, forcing users to scroll a lot to see what options are on the whole page.

#### Suggested styling changes
Please implement the following changes (this will also reduce the height of the card images on the Case Studies page, which is OK):

```
.ant-card-hover img {
	margin: auto;
	min-height: 180px; /* this line is unecessary if the current definition of min-height is deleted */
	max-height: 180px;
	width: auto;
}

cardIllustraton {
       padding: 24px 0 0;
}
```

**Comment by nmathava** [comment]
Mark please enter the card size

**Comment by thegentlemanphysicist** [comment]
@mark-a-wilson  I mostly ignored the suggested CSS because directly overwriting the ant design styling for the entire site is something we moved away from with styled components.  Please let me know if my current fix meets specifications.

**Comment by mark-a-wilson** [comment]
Good improvement. This issue will be fixed by #651 when that PR is merged.
